types/index.d.ts

```diff
@@ -5128,4 +5213,6 @@ declare namespace math {
   interface ImportObject {
     [key: string]: any;
   }
+
```

Comment:
Please revert whitespace-only change (insertion of two blank lines) here, unless there's an argument for why adding one blank line is important (but I don't think this codebase uses two blank lines anywhere).